st2common/st2common/transport/bootstrap_utils.py

```diff
@@ -69,3 +72,12 @@ def wrapped_register_exchanges(connection, channel):
                                       retry_wrapper=retry_wrapper)
 
         retry_wrapper.run(connection=conn, wrapped_callback=wrapped_register_exchanges)
+
+
+def register_exchanges():
+    retrying_obj = retrying.Retrying(
```

Comment:
For consistency with `db_setup` can you please add `register_exchanges_with_retry` method which calls this one with `retrying` stuff?

Thanks!